alert_group_nl_log2slack.py

```diff
@@ -383,18 +390,24 @@ def fetch_logs_and_publish_forever():
     already_published = set()
 
     while True:
+        date = datetime.datetime.now()
         data = set(fetch_logs_with_retry())
         not_published_yet = (data - already_published)
         print(f'data count: {len(data)}, new: {not_published_yet}')
         already_published = data
 
-        a_while_ago = (datetime.datetime.now() - datetime.timedelta(hours=4))
+        a_while_ago = (date - datetime.timedelta(hours=4))
```

Comment:
How is this an improvement?

Reply:
(Ah, because of the date used below. Which isn't needed if using `os.utime()`. In that case I prefer the variable `now()` over `date` because (a) date sounds like days not seconds, and (b) date is also a class.)

Reply:
Not imported here I guess. But it could be:
```pycon
>>> from datetime import date, time
>>> date.today()
datetime.date(2024, 12, 17)
```